test/e2e/fleetautoscaler_test.go

```diff
@@ -20,7 +20,8 @@ import (
 	"testing"
 	"time"
 
-	"agones.dev/agones/pkg/apis/stable/v1alpha1"
+	"agones.dev/agones/pkg/apis/autoscaling/v1alpha1"
```

Comment:
Can you keep this consistent throughout the tests? Like stablev1alpha1 and autoscalingv1alpha1?

Reply:
yeah, that's a fair point :+1: I'll jump on it.

Right now it's a bit "v1alpha1" is the import if that's the most specific thing you are working on (for better or worse), and I think it's mostly like that in a lot of the codebase.

Reply:
I'll merge this change to make sure this gets in for v0.11 release. We can fix this in future PRs.